rust/lance-encoding/benches/decoder.rs

```diff
@@ -201,114 +227,125 @@ fn bench_decode_packed_struct(c: &mut Criterion) {
                     &encoded,
                     &FilterExpression::no_filter(),
                     Arc::<DecoderPlugins>::default(),
                     false,
+                    LanceFileVersion::default(),
+                    Some(Arc::new(FileMetadataCache::no_cache())),
                 ))
                 .unwrap();
             assert_eq!(data.num_rows(), batch.num_rows());
         })
     });
 }
 
-#[allow(dead_code)]
 fn bench_decode_str_with_fixed_size_binary_encoding(c: &mut Criterion) {
     let rt = tokio::runtime::Runtime::new().unwrap();
     let mut group = c.benchmark_group("decode_primitive");
-    // generate string column with 10k rows
-    // Currently the generator generates fixed size strings by default
-    // This function will need to be updated once that changes.
-    let string_data = lance_datagen::gen()
-        .anon_col(lance_datagen::array::rand_type(&DataType::Utf8))
-        .into_batch_rows(lance_datagen::RowCount::from(10000))
-        .unwrap();
 
-    let schema = Arc::new(Schema::new(vec![Field::new(
-        "string",
-        DataType::Utf8,
-        false,
-    )]));
-
-    let data = RecordBatch::try_new(schema, string_data.columns().to_vec()).unwrap();
-
-    let lance_schema =
-        Arc::new(lance_core::datatypes::Schema::try_from(data.schema().as_ref()).unwrap());
-    let input_bytes = data.get_array_memory_size();
-    group.throughput(criterion::Throughput::Bytes(input_bytes as u64));
-    let encoding_strategy = default_encoding_strategy(LanceFileVersion::default());
-    let encoded = rt
-        .block_on(encode_batch(
-            &data,
-            lance_schema,
-            encoding_strategy.as_ref(),
-            &ENCODING_OPTIONS,
-        ))
-        .unwrap();
+    const NUM_ROWS: u64 = 10000;
+    // Randomly generated strings are always 12 characters (at the moment)
+    const NUM_BYTES: u64 = NUM_ROWS * 16;
+    group.throughput(criterion::Throughput::Bytes(NUM_BYTES));
+
     let func_name = "fixed-utf8".to_string();
     group.bench_function(func_name, |b| {
+        // generate string column with 10k rows
+        // Currently the generator generates fixed size strings by default
+        // This function will need to be updated once that changes.
+        let string_data = lance_datagen::gen()
+            .anon_col(lance_datagen::array::rand_type(&DataType::Utf8))
+            .into_batch_rows(lance_datagen::RowCount::from(10000))
+            .unwrap();
+
+        let schema = Arc::new(Schema::new(vec![Field::new(
+            "string",
+            DataType::Utf8,
+            false,
+        )]));
+
+        let data = RecordBatch::try_new(schema, string_data.columns().to_vec()).unwrap();
+
+        let lance_schema =
+            Arc::new(lance_core::datatypes::Schema::try_from(data.schema().as_ref()).unwrap());
+        let encoding_strategy = default_encoding_strategy(LanceFileVersion::default());
```

Comment:
I think fixed size binary encoding is not enabled in `LanceFileVersion::default`(Lance 2.0)

Reply:
I'm not sure but I didn't change the benchmark.  We can revisit when we test fixed-size-binary in 2.1